### PR TITLE
Update o2a converter to support Airflow 2

### PR DESCRIPTION
**Pull request by MaksYermak**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,23 +1,23 @@
-apache-airflow==1.10.14 # pyup: ignore
+apache-airflow==2.6.0 # pyup: ignore
```
Comment (kosteev) [inline requirements.txt:1]: Does this requirement mean that this tool will be used only for Airflow 2.6.0?
If so, can we have e.g. some range here?

**Review by kosteev — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (MaksYermak) [reply requirements.txt:1]: No, this tool uses some utils functions and AirflowExeception from the airflow package. On conversion this package nohow influences and users could use this tool for all Airflow 2 sub-verision.

Comment (kosteev) [reply requirements.txt:1]: ok, sg

**Comment by ayy1ma** [comment]
Can we approve and merge this to main, I recently tested few DAGs on Airflow 2.0, It seems to work as desired. Any ETA when the merge can happen? Does the tool come out of deprecation if Airflow 2.0 is supported?

**Comment by michalmodras** [comment]
@ayy1ma We're in the process of testing and fixing some edge cases. We plan to submit this in the upcoming weeks, and therefor release a new major version of the tool (supporting Airflow 2.x) that will be supported. The version of the tool supporting Airflow 1.x will continue to be deprecated.

**Comment by MaksYermak** [comment]
These changes were merged into the `main` branch.

**Pull request closed without merging**